include/eve/detail/function/simd/common/broadcast.hpp

```diff
@@ -0,0 +1,42 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright 2020 Joel FALCOU
+  Copyright 2020 Jean-Thierry LAPRESTE
+
+  Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/abi.hpp>
+#include <eve/pattern.hpp>
+#include <eve/forward.hpp>
+
+namespace eve::detail
+{
+  template<simd_value Wide>
+  EVE_FORCEINLINE auto broadcast_(EVE_SUPPORTS(cpu_), Wide v, auto Index) noexcept
+  {
+    return v[broadcast_n<Index,cardinal_v<Wide>>];
+  }
```

Comment:
Hmm?
Is this a lookup fallback? Can we please not use the same `operator[]` with very very similar parameters at least in the implementation?

Also this is a questionable CPU impl.

Let's do Wide(v.get(Index)).

Ho is this suppose to change the ABI (which can happen according to generic shuffle)

Reply:
No. lookup is never call from there

Reply:
Fixed